x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ApiKeyService.java

```diff
@@ -1880,7 +1880,7 @@ AtomicLong getLastEvictionCheckedAt() {
      * @return realm name
      */
     public static String getCreatorRealmName(final Authentication authentication) {
-        if (authentication.isApiKey()) {
```

Comment:
The alternative is to use `_es_remote_access` as the realm for remote access **but** it seems more useful from an audit perspective to include the creator realm. This allows us to uniquely identify the remote access API key creator, whereas `_es_remote_access` in and of itself does not provide extra info (an audit entry with a `remote_access` field already tells us a log event is for a remote access authentication). An analogy I see here is with regular API key entries: we use the creator realm instead of `_es_api_key`.